packages/adapter-drizzle/src/index.ts

```diff
@@ -223,22 +261,26 @@ import type { Adapter } from "@auth/core/adapters"
  *    compoundKey: primaryKey({
         columns: [account.provider, account.providerAccountId],
       }),
+      userIdIdx: index('Account_userId_index').on(account.userId)
  *  })
  * )
  *
  * export const sessions = sqliteTable("session", {
- * sessionToken: text("sessionToken").notNull().primaryKey(),
+ * id: text("id").primaryKey().$defaultFn(() => randomUUID())
+ * sessionToken: text("sessionToken").notNull().unique(),
  * userId: text("userId")
  *   .notNull()
  *   .references(() => users.id, { onDelete: "cascade" }),
  * expires: integer("expires", { mode: "timestamp_ms" }).notNull(),
- * })
+ * }, (table) => ({
+ *  userIdIdx: index('Session_userId_index').on(table.userId)
+ * }))
```

Comment:
this is also a breaking change for the underlying database - why was this made?

Reply:
Agree with you. It was a mistake to add indexes for foreign keys. I will remove them in new PR

Reply:
@realmikesolo thanks for stepping up to remove those, feel free to tag me in the PR so we can get it merged quickly :pray: